.github/workflows/upload-release-asset.yml

```diff
@@ -0,0 +1,44 @@
+# Adapted from the example workflow at
+# https://github.com/actions/upload-release-asset
+
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+    - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
```

Comment:
The idea here is to version this separately from the macOS versions right? So v0.1 gets an artifact for 10.12, 10.13, etc.